sandbox-server/app.js

```diff
@@ -51,4 +51,10 @@ app.get('/somefile', (request, response) => {
   }
 })
 
+app.get('/old-page', (request, response) => {
+  response.status(301)
+  response.setHeader( 'Location', 'http://localhost:3000/newpage')
+  response.send()
```

Comment:
```
- [X] Sending a `GET` request to the path `/old-page` responds with...
  - [X] a 301 (Moved Permanently) status code
  - [X] the `Location` header set to `http://localhost:3000/newpage`
```

However, what you're saying does make sense. Thanks for the learning opportunity, we are going with what the spec says.